### ISSUE TITLE
Allow to send and receive data from the same COM port

### ISSUE THREAD

**Issue by max-privato** [modelica-3rdparty/Modelica_DeviceDrivers#350]
To use serial communication is very useful to be able to send and receive data from the same COM port.
Unfortunately, this seems impossible with the current version of Modelica_DeviceDrivers, since the SW tries to open the same port twice. 

See for example the following model (change txt into mo before using):
[RXTXissue.txt](https://github.com/modelica-3rdparty/Modelica_DeviceDrivers/files/7404402/RXTXissue.txt)

This comes from the choice to have separate serialPortSend and serialPortReceive, which both open (or try to open) their ports.
Maybe one should add a serialPortSendReceive with two connectors one to send, the other to receive?

**Comment by bernhard-thiele** [comment]
Yes, or a similar approach as is used for some other devices there where is an inner `config` component which opens the device and the other read/write blocks can use it as `outer`. 

I agree that it would be a nice feature. However, I'm not very knowledgeable regarding serial port programming, particularly on the Windows side. So I guess it would take me quite some time to get things running for both platforms. If somebody would provide a solid Windows implementation I'd be fine with trying for Linux :)
